docs/releasenotes/release-1.126.md

```diff
@@ -17,13 +17,14 @@ TODO: list contributors with `git log v1.125.0... | grep Merge | grep from | awk
 
 ## New Fields:
 
-* `Placeholder`
+* `GkeHubFeatureMembership`
+*   Added `spec.configmanagement.management` field to enable Config Sync Auto Upgrade.
 
 ## Modified Beta Reconciliation
 
 We migrated the following reconciliation from the TF-based or DCL-based controller to the new Direct controller to enhance the reliability and performance. The resource CRD is unchanged.
 
-* `Placeholder`
+* `GkeHubFeatureMembership` is now a direct resource
```

Comment:
oh, we have a new format. let me update the template so we can further broadcast this resource!